common/djangoapps/util/json_request.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, resp_obj=None, status=None, encoder=EDXJSONEncoder,
         elif isinstance(resp_obj, QuerySet):
             content = serialize('json', resp_obj)
         else:
-            content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=False)
+            content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=True)
```

Comment:
@nedbat: Please see the first link that @efischer19 posted above as well.  It shows how json.dumps() outputs invalid JSON when mixing `ensure_ascii=False` and markupsafe's `Markup`.  Can you see any reason why we couldn't change all of these to `ensure_ascii=True`?